hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/audit/AWSRequestAnalyzer.java

```diff
@@ -66,100 +69,104 @@ public class AWSRequestAnalyzer {
    * @return information about the request.
    * @param <T> type of request.
    */
-  public <T extends AmazonWebServiceRequest> RequestInfo analyze(T request) {
+  public RequestInfo analyze(SdkRequest request) {
 
     // this is where Scala's case statement would massively
     // simplify life.
     // Please Keep in Alphabetical Order.
```

Comment:
Hmmmm.... "Keep in alphabetical order" 😄 Given some operations were renamed, we're not doing this at the moment.

Maybe we should re-order these eventually